streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestUtil.java

```diff
@@ -75,7 +75,7 @@ public void process(final Object key, final Object value) {
 
                     @Override
                     public void close() {
-                        System.out.printf("Close processor for task %s", context().taskId());
+                        System.out.printf("Close processor for task %s%n", context().taskId());
```

Comment:
Oops! I noticed the lack of a newline in the output. It didn't matter for the tests because the greps aren't bounded by line.